draft-ietf-core-sid.md

```diff
@@ -661,16 +662,46 @@ implementations needing stable SID values, early allocation as defined in
 
 ### Publication of the ".sid" file {#publink}
 
-For a YANG module approved for publication as an RFC,
-a ".sid" file SHOULD be included in the Internet-Draft as a source code block.
+During publication of an RFC, IANA contacts the designated expert team
+("the team"), who are responsible for delivering a final SID file for
+each module defined by the RFC.
+For a type-3 developer ({{parties-roles}}), the team
+creates a new SID file from each YANG module, see below.
+For a type-2 developer, the team first obtains the existing draft SID
+file from a stable reference in the approved draft; for a type-1
+developer, the team extracts the SID file from the approved draft.
+
+The team uses a tool to generate a final SID file from each YANG
+module; the final SID file has all SID assignments set to "stable" and
+the SID file status set to "published".
+A published ".sid" file MUST NOT contain SID assignments with an
+unstable status.
+
+For the cases other than type-3, the team feeds the existing draft SID
+file as an input to the tool so that the changes resulting from
+re-generation are minimal.
+In any case, the team checks the generated file, including for
+validity as a SID file, for consistency with the SID range
+allocations, for full coverage of the YANG items in YANG module, and
+for the best achievable consistency with the existing draft SID file.
+
+The designated experts then give the SID file to IANA to publish into
+the YANG SID Registry ({{ietf-sid-registry}}) along with the YANG
+module.
+
+The ".sid" file MUST NOT be published as part of the RFC: the IANA
+Registry is authoritative and a link to it is to be inserted in the RFC.
+(Note that the present RFC is an exception to this rule as the SID
+file also serves as an example for exposition.)
+RFCs that need SIDs assigned to their new modules for use in the text
+of the document, e.g., for examples, need to alert the RFC editor in
+the draft text that this is the case.
```

Comment:
Where? How?  Is this the RFC editor or IANA that really needs the heads up?  It seems that we know how to do this with TBD already.  It seems like the right place is in the IANA Considerations, that a SID range allocation request is made.

```suggestion
RFCs that need SIDs assigned to their new modules for use in the text
of the document, e.g., for examples, need to alert the RFC editor in
the draft text that this is the case.
Examples should be generated with SIDTBDxx as the SID number, with xx being the offset into the assigned SID range.
In cases where examples are in serialized formats, such as CBOR, the authors may need to revise those examples once the SID allocation has been made.  
If this is not done via Early Allocation of a SID range, then it MUST be done during AUTH48.
```

Reply:
I'm not sure the SIDTBDXX approach works too well, but it clearly is one way of alerting the RFC editor.
In the CBOR context, we like the examples to actually parse, and SIDTBDXX often won't.
I don't think the document should discuss procedural steps of the RPC such as AUTH48.
It is clear that the RFC editor needs to do "something", so we don't need to go into that level of (brittle) detail.